app/components/templates/store/StoreFAQ.tsx

```diff
@@ -1,7 +1,12 @@
 import type React from 'react'
+import { useEffect } from 'react'
 import FAQSection from '~/components/organism/FAQSection'
 
-const faqs = [
+export function stripHtml(html: string): string {
+	return html.replace(/<[^>]*>?/gm, '')
```

Comment:
For anyone revisiting in the future (possibly me), this was used to strip the children of html content we setup for noscript since faqsection is lazy loaded, but we wanted to still consider seo. we are not using user input for this, this is a hard-coded cleanup.